Re-sync WPT tests for pointer events

lilyspiniolas opened the pull request:
#### fc9a23851031b923769cf1642c8cb289d511fcf3
<pre>
Re-sync WPT tests for pointer events
<a href="https://bugs.webkit.org/show_bug.cgi?id=299625">https://bugs.webkit.org/show_bug.cgi?id=299625</a>
<a href="https://rdar.apple.com/161428181">rdar://161428181</a>

Reviewed by Abrar Rahman Protyasha and Tim Nguyen.

Resync pointerevents directory.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/3d29e5a2a14777cdf35c53820baf14a04d3ecf72">https://github.com/web-platform-tests/wpt/commit/3d29e5a2a14777cdf35c53820baf14a04d3ecf72</a>

* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_is_a_pointerevent.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_is_a_pointerevent_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.https.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-propagates-when-target-is-video_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-propagates-when-target-is-video_touch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate-expected.txt: Added.
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/300788@main">https://commits.webkit.org/300788@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/599fe07dfd523c7fb694d7ab2408a5b10369f724

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/123855 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/43570 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/34267 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/130641 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/76006 "Built successfully") | [❌ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/590e86cd-5f47-4452-9da4-01b2451b5540) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/44294 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/52164 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/94198 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/62505 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/299fa13c-fd34-4e3d-8719-8dde83e3d3cc) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/126808 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/35283 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/110799 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/74796 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/5d02faae-d8c5-4db4-9d8f-d24b9ab83c91) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/34239 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/28960 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/74118 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/105010 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/29183 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/133335 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/50808 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/38689 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/102665 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/51182 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/107019 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/102494 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/26062 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/47833 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/26096 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/47657 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/50661 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/50135 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/53481 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/51809 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/6c923153973600e85073a2a7ce43591bdbeffcf4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51403

aprotyas commented on LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-propagates-when-target-is-video_touch.html:
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Does this test not time out on iOS?

aprotyas replied:
oh, of course, all pointerevents WPTs are skipped

aprotyas reviewed (approved):
Can you add which upstream WPT commit this synced against in the commit message? The importer leaves some output you can key off of, something like this:


```
Upstream commit: https://github.com/web-platform-tests/wpt/commit/...
```

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b01c779257b29a21e1fffa2902790574b261a363. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51403

lilyspiniolas committed:
Re-sync WPT tests for pointer events
https://bugs.webkit.org/show_bug.cgi?id=299625
rdar://161428181

Reviewed by Abrar Rahman Protyasha and Tim Nguyen.

Resync pointerevents directory.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3d29e5a2a14777cdf35c53820baf14a04d3ecf72

* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_is_a_pointerevent.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_click_is_a_pointerevent_mouse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate.https.html:
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-propagates-when-target-is-video_touch-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_touch-propagates-when-target-is-video_touch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/pointerevents/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointerrawupdate-expected.txt: Added.
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/300788@main

webkit-commit-queue commented:
Committed 300788@main (fc9a23851031): <https://commits.webkit.org/300788@main>

Reviewed commits have been landed. Closing PR #51403 and removing active labels.
